src/Connection.cpp

```diff
@@ -29,6 +29,7 @@ Connection::Connection(QObject *p, QSslSocket *qtsSock) : QObject(p) {
 	qtsSocket->setParent(this);
 	iPacketLength = -1;
 	bDisconnectedEmitted = false;
+	csCrypt = std::make_unique<CryptStateOCB2>();
```

Comment:
The Linux CI Fails here as apparently [make_unique](https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique) was introduced in cpp14 and is thus not available in cpp11 (which is what we have set for the project).

I think we have 3 options here:
1. Go back to how you initially assigned the pointer
2. Upgrade our minimum cpp-standard to cpp14
3. Use Boost (https://www.boost.org/doc/libs/1_73_0/libs/smart_ptr/doc/html/smart_ptr.html#make_unique)

Personally I don't like 1) and as I think that we should probably consider updating the cpp standard anyways (at latest at the time we want to start refactoring/rewriting), my personal preference would be 2)

Reply:
davide just approved moving to cpp14 (which will be what the cmake branch will introduce anyways).

I'll set this in a separate PR. Once that is merged, you can pull the current master branch (from this repo) into the local branch on your repo in order to include this commit as well. This should then resolve the CI failure :)